### ISSUE TITLE
RuntimeWarning: invalid value encountered in true_divide

### ISSUE THREAD

**Issue by razeineldin** [googlecolab/colabtools#825]
I am training a deep learning model using Keras. Then suddenly during the training, I got this error
/usr/local/lib/python3.6/dist-packages/ipykernel_launcher.py:36: RuntimeWarning: invalid value encountered in true_divide
After that, all training values (like loss and dice, ... etc) change dramatically to wrong values (loss increases, accuracy decrease to 0 'for next epochs').

Epoch 24/50: loss: 0.1081 - dice: 0.7011/usr/local/lib/python3.6/dist-packages/ipykernel_launcher.py:36: RuntimeWarning: invalid value encountered in true_divide
loss: 0.2068 - dice: 0.5500

When I stop running the cell, I got this error: ERROR:root:Internal Python error in the inspect module. as well as the google drive is not mounted any more.

Please note that:
- My web browser is Chrome running on an Ubuntu laptop.
- Initially, the training weights are loaded from my google drive.

Thanks for your help.

Updates:
1- I trained the model for 24 epochs without any problem before the above one arose. 
2- It seems to be a problem in the dataset stored at the Colab because it becomes empty during the training and then the problem is encountered in true_divide. So, the problem now is why the Colab delete my stored dataset during the training although the disk storage is not full?

**Comment by razeineldin** [comment]
Actually this problem is related to my code. I am surprised as it run smoothly for 24 epochs!!
I solved by this link:
https://www.kaggle.com/questions-and-answers/53053

**Comment by nehagupta0293** [comment]
import numpy as np
import pandas as pd
import csv 
from pgmpy.estimators import MaximumLikelihoodEstimator
from pgmpy.models import BayesianModel
from pgmpy.inference import VariableElimination
from pgmpy.metrics import correlation_score


data_training = pd.read_csv('dataset_bn training 70 100 copy.csv')
data_testing = pd.read_csv('dataset_bn testing 30 30 copy.csv')
#heartDisease = heartDisease.replace('?',np.nan)

print('Sample instances from the dataset are given below')
print(data_training.head())

print('\n Attributes and datatypes')
print(data_training.dtypes)

model= BayesianModel([('happy','emotion_label'),('sad','emotion_label'),('angry','emotion_label')])
print('\nLearning CPD using Maximum likelihood estimators')
model.fit(data_training,estimator=MaximumLikelihoodEstimator)




correlation_score(model, data_testing, test="chi_square", significance_level=0.05)



output
unfile('/Users/nehagupta/pythonfiles/finalcode.py', wdir='/Users/nehagupta/pythonfiles')
Sample instances from the dataset are given below
      happy       sad     angry  emotion_label
0  0.620048  0.162409  0.215713            1.0
1  7.745901  0.968815  0.058426            1.0
2  0.982792  0.010134  0.006743            1.0
3  0.204016  0.143374  0.005915            1.0
4  2.607523  0.447393  0.429940            1.0

 Attributes and datatypes
happy            float64
sad              float64
angry            float64
emotion_label    float64
dtype: object

Learning CPD using Maximum likelihood estimators
/Users/nehagupta/opt/anaconda3/lib/python3.9/site-packages/pgmpy/models/BayesianModel.py:8: FutureWarning: BayesianModel has been renamed to BayesianNetwork. Please use BayesianNetwork class, BayesianModel will be removed in future.
  warnings.warn(
/Users/nehagupta/opt/anaconda3/lib/python3.9/site-packages/pgmpy/factors/discrete/CPD.py:332: RuntimeWarning: invalid value encountered in true_divide
  tabular_cpd.values = (cpd / cpd.sum(axis=0)).reshape(tabular_cpd.cardinality)


what is the issue and how I resolve please tell me
